### ISSUE TITLE
Flags used for `ReadDirectoryChangesW` are very aggressive

### ISSUE THREAD

**Issue by bpasero** [libuv/libuv#3403]
* **Version**: 1.40
* **Platform**: Windows

The flags used for `ReadDirectoryChangesW` are imho unusually verbose:

https://github.com/libuv/libuv/blob/bb6248bca4ccc4cb50b3e859be23afd9be52ebcb/src/win/fs-event.c#L47-L54

Compare with the set of default filters the .NET framework decides to pick:

https://github.com/microsoft/referencesource/blob/5697c29004a34d80acdaf5742d7e699022c64ecd/System/services/io/system/io/FileSystemWatcher.cs#L46

I think what libuv configures here has a higher risk of reporting changes on conditions that are unexpected for an upstream file watcher, such as when a file was last accessed.

**Comment by bnoordhuis** [comment]
Absent the presence of real bugs I'm disinclined to change the current flags, simply because that's how libuv has been doing it for a decade. It's very likely downstream code relies on the current behavior.

**Comment by bpasero** [comment]
Indeed https://github.com/libuv/libuv/commit/1e0757ffda47c625b0f7feeeb91c399864787f9e

**Comment by cjihrig** [comment]
I agree with Ben. I'll go ahead and close this out. Thanks for the issue anyway.

**Comment by bpasero** [comment]
I am also a Ben ;-)
